tests/integration/bootstrap.sh

```diff
@@ -27,11 +27,20 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
+## This file bootstrap dependencies of integration tests.
```

Comment:
why double `##`?

Reply:
Not really necessary, just to mimic c-like languages' single line comment(`//`).